### PR TITLE
Update of CONTRIBUTING

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
I changed a few typos and consistently changed "the concepticon" to "the **C**oncepticon". I added the beginners guide and @tresoldi blog posts and left some comments for possible changes.

@chrzyki Please have a look at this first.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,41 +90,43 @@ ID	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	SIMILARITY
...
 7 Have you tested your modifications by typing ```$ concepticon test``` in the command line?
-8 Have you computed the new statistics, by typing ```$ concepticon stats``` and commited the files [```concepticondata/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/README.md) and [```concepticondata/conceptlists/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists/README.md)?
+8 Have you computed the new statistics, by typing ```$ concepticon stats``` and commited the files [```concepticondata/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/README.md) and [```concepticondata/conceptlists/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists/README.md)? (AT: Might be deleted due to our new convention of updateing after around 10 newly added lists)
```
Comment (LinguList) [inline CONTRIBUTING.md]: Yes, @AnnikaTjuka, this does no longer belong here, as it is part of the release procedure and complicates the review process, so we need to add this in the release check list, not in the PR check list.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,41 +90,43 @@ ID	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	SIMILARITY
...
-5 Do you reference a source file in PDF, in your description of the concept list in [```conceptlists.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists.tsv)? If so, have you added it (in the form ```FirstAuthorLastNameYear```) to the folder [```sources/```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/sources/)?
-6 Do you have to thank some person for help with the concept list? If you are contributing and not part of our core editorial team, we will add you, so please provide us with your name and affiliation. To keep track of whom to thank, please edit the [```CONTRIBUTORS.MD```](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTORS.md) document.
+5 Do you reference a source file in PDF, in your description of the concept list in [```conceptlists.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists.tsv)? If so, have you added it (in the form ```FirstAuthorLastNameYear```) to the folder [```sources/```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/sources/)? (AT: Might also need editing.)
```
Comment (LinguList) [inline CONTRIBUTING.md]: Basically, this is still required and should be done, but recently, we had many lists that were too long (if the source has 500 pages, it is not good to add it), but in many cases, one could still do so or one could HAVE done so, as in the color terms, for example, where a simple screenshot converted to PDF would to the trick.

Basically, I prefer using source files and I created them at times deliberately, even by making many screenshots or the like, as it allows to check and type off data more properly, but I would say: this requirement is weaker now, with the many lists we have added, yet it should still remain here.

**Comment by LinguList** [comment]
Ah, yes: so if a person makes a PR, they provide above checklist. Then, one of the acting editors decides how to *classify* this PR, and assigns reviewers. The reviewer assignment is dependent on who is available, e.g., if one is in holiday, we don't assign them. Acting editors should be aware that more complex changes require more people to be involved, and also more time for checking. The reviewer who then manages the PR, the "managing reviewer" from our editorial board, will also be the person to close an issue. 

Every editor has the right to be managing reviewer, but the one who makes the PR is usually NOT the managing reviewer. So the one who makes a PR can assign one person to suggest that this person becomes the acting reviewer. 

This may sound complicated, but I prefer it over being assigned to all issues by default. I think, the whole process has matured enough now, that you guys can add new lists without me reviewing all of them. But in some processes, I still want more people involved.

@AnnikaTjuka, @Kristina-Pianykh, @Schweikhard, @MacyL, as newbies, what do you think about this?

**Comment by Kristina-Pianykh** [comment]
May I paraphrase it to make sure I understood it correctly?
The order of assigning reviewers when making a PR is the following:

1. A person opening a PR chooses ONE acting reviewer from the current editors.

2. This acting reviewer, in their turn, evaluates how to classify the PR and assigns either more than half of the Concepticon's editors (lots of changes) or 2-3 of them (like a basic concept list).

3. Upon making a decision, the acting reviewer merges the PR? and closes the corresponding issue.

Are there already any conventions on how many approvals should there be to merge in each case?

**Comment by LinguList** [comment]
Yes, @Kristina-Pianykh, this is what I meant. The number of approvals
required is the number of reviewers assigned. By now, we are not
consequent in this regard, with the new schema, we can, as we will have
fewer reviewers for smaller issues.

**Comment by chrzyki** [comment]
@LinguList: With PR templates you can prepare a template for pull requests that automatically appends a markdown file, like so: https://github.com/concepticon/concepticon-data/blob/master/.github/pull_request_template.md

This would get added as text to new pull requests. You think that's helpful?

**Comment by LinguList** [comment]
If the text is modifiable, this would be extremely helpful,. @chrzyki.

**Comment by LinguList** [comment]
and yes of course: it is modifiable, since it is a checklist.

**Comment by LinguList** [comment]
Any more points we should discuss or add here, in the template?

**Comment by chrzyki** [comment]
Yes, and the markdown can be further refined here: https://github.com/concepticon/concepticon-data/blob/master/.github/pull_request_template.md

It is also possible to create more fine grained templates, but this might be overkill: https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

**Comment by LinguList** [comment]
A simple one will do, I think.

**Comment by chrzyki** [comment]
I've also update a little bit, feel free to check @AnnikaTjuka.

**Comment by LinguList** [comment]
Hehe, who's the working reviewer for this PR now? In my opinion, you can merge this directly. We should ideally finish all remaining conceepticon issues today, so we can release tomorrow or even later today...

**Comment by AnnikaTjuka** [comment]
@chrzyki Thanks for editing the things I pointed out. I have no further remarks.

**Pull request merged**